.github/workflows/tests.yml

```diff
@@ -44,6 +44,6 @@ jobs:
 
                 set -euo pipefail
 
-                richgo test ./...
+                GOCMD=richgo make
```

Comment:
Sorry I lost this comment on the original review. This tests are managed by https://github.com/paketo-buildpacks/pipeline-builder. In order to prevent the pipeline builder from continually trying to undo this change, we need to [configure `test`]rhttps://github.com/paketo-buildpacks/pipeline-builder#test) in https://github.com/buildpacks/libcnb/blob/main/.github/pipeline-descriptor.yml. Or, move away from pipeline builder entirely.

Reply:
Or convince @dmikusa-pivotal to add a `Makefile` to all the Paketo buildpacks that use pipeline builder and update the default `test` config ;). Which seems like a good idea honestly, but is probably a lot to ask him to do today.

Reply:
Oh wow - did not know about this. Added the configuration for now :P I don't wanna put more on @dmikusa-pivotal's plate.

Reply:
I wouldn't be opposed to parameterizing it. It would be a small change, something like `TEST_CMD`. The template it creates would run `$TEST_CMD` then. I could also see parameterizing GOCMD, not everyone might want to use `richgo`. Then you wouldn't need to set GOCMD, it would just be set by the workflow based on the pipeline descriptor values. If that seems reasonable, I can open an issue to make the change.

Reply:
> I wouldn't be opposed to parameterizing it. It would be a small change, something like TEST_CMD.

Personally I think overriding the whole step is fine in this situation.

Reply:
Sorry @ekcasey, I'm not sure I follow what you mean. It's OK though, if you or @samj1912 want to submit an issue with details of what you need to https://github.com/paketo-buildpacks/pipeline-builder/issues, I'll make it happen for you.